my-app/src/App/pages/ProductPage/components/InfoProductCard/InfoProductCard.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useState } from "react";
+
+import Button from "@components/Button";
+import Carousel from "@components/Carousel";
+import { API_ENDPOINT } from "@config/api";
+import axios from "axios";
+import { useParams } from "react-router-dom";
+
+import styles from "./InfoProductCard.module.scss";
+import RelatedProducts from "../RelatedProducts";
+
+const InfoProductCard = () => {
+  const [card, setCard]: any = useState({});
```

Comment:
any лучше не использовать вообще, потому что тогда теряется смысл вообще в использовании типизации
В данном случае указывать тип у массива не нужно, потому что туда сразу кладется значение. Нужно было указать для хука какой использовать дженерик, т.е. 
```suggestion
  const [card, setCard] = useState<SomeCardProps>({});
```

Reply:
и нужно было описать тип объекта, которые ты получаешь с бэка